ut/s3_get_object_action_test.cc

```diff
@@ -1202,3 +1202,57 @@ TEST_F(S3GetObjectActionTest, RangeHeaderContainsSpacesOnly) {
       range_value));
 }
 
+TEST_F(S3GetObjectActionTest, ReadObjectOfSizeEqualToUnitSizeMD5Fail) {
```

Comment:
Need similar test which validates code that handles multipart objects.

Reply:
multipart test added